TestAssets/TestProjects/docker-compose/docker-compose.dcproj

```diff
@@ -0,0 +1,23 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" Sdk="Microsoft.Docker.Sdk">
```

Comment:
According to https://github.com/Microsoft/msbuild/issues/699#issuecomment-243934939, it is ignored by MSBuild.

/cc @rainersigwald @jeffkl @cdmihai